### PR TITLE
AzureStack.Admin.Compute SDK 2018-02-09 compute quota update

### PR DESCRIPTION
**Pull request by cheguv**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by cheguv** [comment]
All tests have passed with these changes with newly recorded sessions.
I did not see any reference to the version info in *.csproj or AssemblyInfo.cs files to update for sdk version.
Please let me know if there is any thing required from my side.

**Comment by bganapa** [comment]
version infor is here
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/AzureStack/Admin/ComputeAdmin/Compute.Admin/Microsoft.AzureStack.Management.Compute.Admin.csproj#L10

https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/AzureStack/Admin/ComputeAdmin/Compute.Admin/Properties/AssemblyInfo.cs#L17

I would suggest 0.2.0-preview

**Comment by bganapa** [comment]
Please link the swagger spec PR here

**Comment by cheguv** [comment]
updated the nuget and assembly version.

Swagger spec link: https://github.com/Azure/azure-rest-api-specs/pull/3564

**Comment by bganapa** [comment]
Also pls update the PackageReleaseNotes in the csproj, Add the changes from 0.1.0 to 0.2.0, i recommend mentioning the api-version change and the addition of properties

File: src/AzureStack/Admin/ComputeAdmin/Compute.Admin/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -14,5 +14,5 @@
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en")]
 
-[assembly: AssemblyVersion("0.1.0.0")]
-[assembly: AssemblyFileVersion("0.1.0.0")]
+[assembly: AssemblyVersion("0.2.0.0")]
+[assembly: AssemblyFileVersion("0.2.0.0")]
```
Comment (deathly809) [inline src/AzureStack/Admin/ComputeAdmin/Compute.Admin/Properties/AssemblyInfo.cs]: Don't change this!

**Review by bganapa — APPROVED** [review]

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please regenerate the code using `generate.ps1` [here](https://github.com/Azure/azure-sdk-for-net/blob/d97b6cd231f54dce65c0dc7f2b38dce8930f18a0/src/AzureStack/Admin/ComputeAdmin/Compute.Admin/generate.ps1) and commit all the changes generated (including log .txt files)

File: src/AzureStack/Admin/ComputeAdmin/Compute.Admin/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (deathly809) [reply src/AzureStack/Admin/ComputeAdmin/Compute.Admin/Properties/AssemblyInfo.cs]: @dsgouda Should this file be changed?  I have heard conflicting information!

Comment (dsgouda) [reply src/AzureStack/Admin/ComputeAdmin/Compute.Admin/Properties/AssemblyInfo.cs]: For a version bump yes.
The guideline is:
If a major version is bumped, i.e., from `0.0.0` to `1.0.0`, both `AssemblyVersion` and `AssemblyFileVersion` must be changed.
If a minor/patch version is bumped, only the `AssemblyFileVersion` must be bumped, i.e., `0.1.0` to `0.2.0`
tagging @shahabhijeet

Comment (deathly809) [reply src/AzureStack/Admin/ComputeAdmin/Compute.Admin/Properties/AssemblyInfo.cs]: Thanks for the clarification!

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge on CIs passing

**Pull request merged**